apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/pagerules/PageRuleControllerTest.java

```diff
@@ -80,5 +86,58 @@ public void shouldReadRule() {
         ViewRuleResponse ruleResponse = pageRuleController.viewRuleResponse(ruleId);
         assertNotNull(ruleResponse);
     }
+    @Test
+    void shouldReadAllRule() throws Exception {
```

Comment:
Controllers are bettered tested using the integration tests run with cucumber.   These tests just make more work if the service is changed